cql3/Cql.g

```diff
@@ -1169,6 +1170,7 @@ listPermissionsStatement returns [std::unique_ptr<list_permissions_statement> st
     ;
 
 permission returns [auth::permission perm]
+    @init { $perm = auth::permission{}; }
```

Comment:
Unless I'm missing something, auth::permission is an enum, what does this empty initialization even mean?

Again, since the only rule assigns $perm, it looks like either an antlr bug (that can return without a rule? how?) or a compiler bug, and I'd be happy to know which... If we believe it's a compiler bug, maybe we should add a comment saying that, so that future readers don't think it's needed for some important reason.